src/fileio.cpp

```diff
@@ -1013,7 +1014,7 @@ size_t sphReadThrottled ( int iFD, void* pBuf, size_t iCount )
 	if ( iCount <= 0 )
 		return iCount;
 
-	auto iStep = g_iMaxIOSize ? Min ( iCount, (size_t)g_iMaxIOSize ) : iCount;
+	auto iStep = g_iMaxIOSize ? Min ( iCount, (size_t)g_iMaxIOSize ) : iCount; // Now always true
```

Comment:
If it is always true, m.b. simplify (throw out) ternary condition and leave just 'always true' part, that is
`Min ( iCount, (size_t)g_iMaxIOSize )` ?

Reply:
will be done